app/components/common/emailChipsInput/index.tsx

```diff
@@ -0,0 +1,112 @@
+// TODO: remove all eslint-disable when eslint-prettier issue is fixed
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-useless-escape */
+import React, { FC, useState } from 'react';
+
+interface EmailChipsInputProps {
+  label: string;
+  placeholder: string;
+  attendees: string[];
+  setAttendees: (att: string[]) => void;
+}
+
+const EmailChipsInput: FC<EmailChipsInputProps> = ({
+  label,
+  placeholder,
+  attendees,
+  setAttendees,
+}) => {
+  const [value, setValue] = useState('');
+  const [error, setError] = useState<string | null>(null);
+
+  const isEmailInList = (email: string) => attendees.includes(email);
+
+  const isEmail = (email: string) => /[\w\d\.-]+@[\w\d\.-]+\.[\w\d\.-]+/.test(email);
```

Comment:
can we move this to utils?

Reply:
@isVivek99 we can if there is another usecase where we will need it. I checked login and signup but we are not checking for email there. We can create a separate issue to extract it from here, add it to utils and implement it in login/signup as well. Or should I address it here itself?